nservicebus/msmq/distributor/configuration_master_core_[,4).partial.md

```diff
@@ -0,0 +1,5 @@
+It is also possible to use the `NServiceBus.Master` profile to start a Distributor on the endpoint with a Worker on its endpoint. To start the endpoint as a Master run the host from the command line, as follows:
```

Comment:
> It is also possible to use the `NServiceBus.Master` profile to start a Distributor on the endpoint with a Worker on its endpoint

I don't understand why "master" profile is starting worker? 

Also what does it mean to "start sth with a Worker on its endpoint"? I think we need to rephrase it.